codespell_lib/data/dictionary.txt

```diff
@@ -11347,6 +11347,10 @@ errror->error
 errrors->errors
 errros->errors
 errupted->erupted
+ertoneous->erroneous
+ertoneously->erroneously
+ertor->error
+ertors->errors
```

Comment:
```suggestion
ertor->error, terror,
ertors->errors, terrors,
```

Reply:
You know you can just edit the files directly as the PR author (even via the web UI if preferred), you don't have to use suggestions. Although fair enough if you find it easier.

Reply:
Ah, yes. Good suggestion :+1: